src/accelerate/utils/other.py

```diff
@@ -14,12 +14,10 @@
 
 import os
 from contextlib import contextmanager
-from pathlib import Path
 
 import torch
 
-from ..commands.config.cluster import ClusterConfig
-from ..commands.config.config_args import default_json_config_file
+from ..commands.config.default import write_basic_config  # noqa: F401
```

Comment:
is this required here anymore?

Reply:
Yep, this is the nit about making sure we can keep `from accelerate.utils import write_basic_config`